src/Command/Adapter/AdapterInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Human Scoring Software
+ *
+ * @author antoinep@taotesting.com
+ * @license See LICENCE.md
+ */
+namespace App\Command\Adapter;
+
+
+interface AdapterInterface
```

Comment:
I think we can find a better name than AdapterInterface. Usually you would prefix this with something as AdapterInterface remains quite vague. Also, in your command, when you inject an AdapterInterface you name it "$dataProvider", so might be worth taking a look.